src/app/components/home/book-page/dnf/dnf.component.html

```diff
@@ -5,51 +5,61 @@ <h2 class="text-xl font-bold uppercase">Deal Breakers</h2>
   >
     <div
```

Comment:
Consider having these badges as a separate component. Then in the ` dnf.component.html ` , you can use a ` @for ` control flow to display all of them. Maybe have an array with the Reasons (Characters, Plot, Writing Style etc. ) to loop over in the ` @for ` control flow